ghost/admin/app/components/posts-list/modals/delete-posts.hbs

```diff
@@ -0,0 +1,28 @@
+<div class="modal-content" data-test-modal="delete-posts">
+    <header class="modal-header">
+        <h1>Are you sure you want to delete {{if @data.isSingle 'this post' 'these posts'}}?</h1>
+    </header>
+    <button type="button" class="close" title="Close" {{on "click" (fn @close false)}} data-test-button="close">{{svg-jar "close"}}<span class="hidden">Close</span></button>
+
+    <div class="modal-body">
+        <p>
+            You're about to delete {{if @data.isSingle 'this post' 'these posts'}}.
```

Comment:
Should we include the number of posts here? cc @sanne-san

Reply:
Yes, that'd be great! Single looks good, for multiple could we say 'You're about to delete `X` posts.' instead?

Reply:
It is possible, but a bit complicated. When doing CMD+A we'll need to know the total amount of posts in the view and that is currently not yet passed to the list. I'll see how much extra effort this is and add this or create an issue for this

Reply:
Was easier than I thought 😅